pkg/partitionerv2/mkfs.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package partitioner
+package partitionerv2
```

Comment:
why can't we leave this one as partitioner ?

Reply:
because Im stupid? 

I had them along together in case we wanted to reverse it :D

Reply:
gotcha, I think if we rollback I'd rather go with rolling back the merged commit, and the only case I'd keep them side by side is if there was a special case and we needed to still use the previous command, or if you were migrating them on batches or something like that

Reply:
done